nixos/modules/services/continuous-integration/concourse/worker.nix

```
@@ -0,0 +1,224 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.concourse.worker;
+  useContainerd = cfg.runtime.type == "containerd";
+  useGuardian = cfg.runtime.type == "guardian";
+in
+{
+  meta.maintainers = with lib.maintainers; [ lenianiva ];
+
+  options.services.concourse.worker = {
+    enable = lib.mkEnableOption "A container-based automation system written in Go. (The worker part)";
```

```suggestion
    enable = lib.mkEnableOption "Concourse worker";
```
Whether to enable ... . will be appended automatically